packages/testcontainers/src/containers/RegTestContainer/LoanContainer.ts

```diff
@@ -4,7 +4,7 @@ import { MasterNodeRegTestContainer } from './Masternode'
 
 export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
   constructor (masternodeKey: MasterNodeKey = RegTestFoundationKeys[0]) {
-    super(masternodeKey, 'defi/defichain:master-da0bbfa')
+    super(masternodeKey, 'defi/defichain:HEAD-c4fc84b')
```

Comment:
huh?? why not refer to the latest??

Reply:
Mine is the latest in `v2.0.x`. As it was merged to master, I shall change it to `master-da0bbfa`